resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -58,37 +58,8 @@ EXEC="$JAVA_BIN -XX:OnOutOfMemoryError=\"kill -9 %p\" -Djava.security.egd=/dev/u
 #
 do_start()
 {
-    <% if EZBake::Config[:open_file_limit] -%>
-    [ -n "$OPEN_FILE_LIMIT" ] && ulimit -n $OPEN_FILE_LIMIT
-    <% end -%>
-
-    <% EZBake::Config[:debian][:pre_start_action].each do |action| -%>
-    <%= action %>
-    <% end -%>
-
-    if [ ! -d  "/var/run/puppetlabs/$NAME" ] ; then
-      mkdir -p /var/run/puppetlabs/$NAME
-      chown -R $USER:$GROUP /var/run/puppetlabs/$NAME
-    fi
-
-    start-stop-daemon --start $EXTRA_ARGS --quiet --oknodo --pidfile $PIDFILE --chdir $INSTALL_DIR --exec $JAVA_BIN \
-      --startas /bin/bash -- -c "exec $EXEC >> /var/log/puppetlabs/${REALNAME}/${REALNAME}-daemon.log 2>&1"
+    /opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> start
     retval=$?
-
-    if [ "$retval" -ne 0 ] \
-        || ! wait_for_pidfile $PIDFILE 5 \
-        || ! wait_for_app $(cat $PIDFILE) $START_TIMEOUT ;then
-        retval=2
-    fi
-
-    <% if not EZBake::Config[:debian][:post_start_action].empty? -%>
-    if [ "$retval" -eq 0 ]; then
-    <% EZBake::Config[:debian][:post_start_action].each do |action| -%>
-        <%= action %>
-    <% end -%>
```

Comment:
See previous comment re: `pre_start_action` hooks, but applied to `post_start_action` instead.